sbtc-core/src/operations/op_return/withdrawal_request.rs

```diff
@@ -449,3 +457,22 @@ pub fn create_signing_message(data: impl AsRef<[u8]>) -> Message {
 	Message::from_slice(Sha256Hasher::new(msg_content).as_ref())
 		.expect("Could not create secp message")
 }
+
+// test that create signing message returns correct hash
+#[cfg(test)]
+mod tests {
+	use super::*;
+
+	#[test]
+	fn test_create_signing_message() {
+		let address: BitcoinAddress =
+			"tb1qwe9ddxp6v32uef2v66j00vx6wxax5zat223tms"
+				.parse()
+				.unwrap();
+		let msg = create_withdrawal_request_signing_message(1000, &address);
+		assert_eq!(
+			msg.to_string(),
+			"744eee0ee13d6649dd6b0fe203d2cb0af32e5d0b57a7c046c782019e8d562056"
```

Comment:
What would be a good source? This issue?

Reply:
Linking the issue sounds good, too, but I was thinking more of a quick note for the reader instead of having to digest a ticket. Your call, I'm okay with either.

Where would you check if you doubted the correctness of the literal? Imagine this test was implemented before applying the proposed changes, and the test is passing. To pass, a different literal value was being asserted. Without context, this literal is a magic value for a regression test; it is not ideal. A regression test should point to the source used as the 'truth.' I assume the value comes from another component (the leather wallet?) and a message stating so would help. I would expect a comment like 'parity with leather wallet's signing output.'

Reply:
I see. I'll add that it is the sha 256 of the message in the format and so on.